config/locales/client.en.yml

```diff
@@ -2377,6 +2377,12 @@ en:
         high_priority:
           one: "%{count} unread high priority notification"
           other: "%{count} unread high priority notifications"
+        new_message_notification:
+          one: "%{count} new message notification"
+          other: "%{count} new message notifications"
+        new_reviewable:
+          one: "%{count} new reviewable"
+          other: "%{count} new reviewables"
```

Comment:
Just thinking out loud here but would adding `pending` help to improve the title?